File: Semester 4/Programming and Development Methods/MPPProject/src/main/java/Domain/Athlete.java
package Domain;

import java.util.Date;

public class Athlete implements IdEntity<Integer> {

    private int id;
    private String name;
    private java.sql.Date birthDate;

    public Athlete(int id, String name, java.sql.Date birthDate) {
        this.id = id;
        this.name = name;
        this.birthDate = birthDate;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Date getBirthDate() {
        return birthDate;
    }

    public void setBirthDate(java.sql.Date birthDate) {
        this.birthDate = birthDate;
    }

    @Override
    public Integer getId() {
        return id;
    }

    @Override
    public void setId(Integer integer) {
        this.id = id;
    }
}

File: Semester 4/Programming and Development Methods/CSProject/Project/Project/Domain/Athlete.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Project.Domain
{
    class Athlete : IdEntity<int>
    {
        public Athlete(int id, string name, DateTime birthDate)
        {
            this.id = id;
            this.name = name;
            this.birthDate = birthDate;
        }

        public int id { get; set; }
        public String name { get; set; }
        public DateTime birthDate { get; set; }



        public int GetId()
        {
            return id;
        }

        public void SetId(int id)
        {
            this.id = id;
        }
    }
}
